pkg/apis/policy/v1alpha1/types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/util/intstr"
+)
+
+// PodDisruptionBudgetSpec is a description of a PodDisruptionBudget.
+type PodDisruptionBudgetSpec struct {
+	// The minimum number of pods that must be available simultaneously.  This
+	// can be either an integer or a string specifying a percentage, e.g. "28%".
+	MinAvailable intstr.IntOrString `json:"minAvailable,omitempty"`
+
+	// Selector is a label query over pods whose evictions are managed by the
+	// disruption budget.
+	Selector *unversioned.LabelSelector `json:"selector,omitempty"`
+}
+
+// PodDisruptionBudgetStatus represents information about the status of a
+// PodDisruptionBudget. Status may trail the actual state of a system.
+type PodDisruptionBudgetStatus struct {
+	// Whether or not a disruption is currently allowed.
+	PodDisruptionAllowed bool `json:"disruptionAllowed"`
+
+	// current number of healthy pods
+	CurrentHealthy int32 `json:"currentHealthy"`
+
+	// desired number of healthy pods
+	DesiredHealthy int32 `json:"desiredHealthy"`
```

Comment:
See https://github.com/kubernetes/kubernetes/pull/24697#issuecomment-215569613.  Search for the string "percentage"